src/components/Colorbox.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+const colors = {
+  items: 28,
```

Comment:
what does the 28 means?

Reply:
The original picture only shows 28 squares.

Reply:
So to answer my own question: It means the number of colors.

This leads me to another question:
Isn’t it the same as the array lenghth? Why duplicating the information?

Reply:
Ping @ylruiz